Phonebook.Frontend/src/app/modules/table/helpers.ts

```diff
@@ -93,7 +93,10 @@ export class Helpers {
    * Removes Accents from the string.
    * @param str Any String
    */
-  public static removeAccents(str: string): string {
+  public static removeAccents(str: string | null): string {
```

Comment:
das verursacht evtl. unbeabsichtigtes fehlverhalten, da `removeAccents` in sortierfunktionen eingesetzt wird. Ich bau es mal so um, dass es trotzdem funktioniert.

Ich hoffe du hast es die nullbaren Dinge in den Models angepasst?

Reply:
set all model properties to nullable.